tensorboard/webapp/runs/views/runs_table/regex_edit_dialog.ng.html

```diff
@@ -41,6 +42,19 @@ <h1 mat-dialog-title>Color runs by regex</h1>
   </p>
 </div>
 
+<div *ngIf="colorRunsMap.length" class="group-container">
+  <div>Grouping result preview:</div>
+  <div class="grouping-preview">
+    <div
+      class="group"
+      *ngFor="let colorRunsPair of colorRunsMap"
+      style="background: {{colorRunsPair[0]}}"
+    >
+      <span *ngFor="let run of colorRunsPair[1]"> {{ run.name }}<br /> </span>
```

Comment:
consider using ul/li for list of things. Also consider truncating the result (and say "and N more runs matches") as seeing 100s of result is not that helpful anyways